apps/registry/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "registry",
-  "version": "1.0.1",
```

Comment:
That is annoying. I find this should not happen. Actually these packages should not have a version at all, they are "apps" after all. I will investigate how to prevent this.